getting_hired/preparing_for_job_search/professional_networking.md

```diff
@@ -0,0 +1,82 @@
+### Introduction
+
+A robust professional network is an irreplaceable asset on your journey to becoming a web developer, and beyond. As social beings we are wired to connect with others, receive help, and give help. This is no different in the software ecosystem where countless communities take part in the mutually beneficial transactions of networking.
+
+One significant benefit of your network is that it might help land you your first job in software. This is not a guarantee, but it is the lived experience of many developers. As with all things, you get back what you put in, so the more developed your network is, the better your odds it will provide for you. 
```

Comment:
might help _**you**_ land ~~you~~ your first

Reply:
good catch